api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -560,6 +565,22 @@ def delete_venue(venue_id: int) -> utils.BackofficeResponse:
     return redirect(url_for("backoffice_web.pro.search_pro"), code=303)
 
 
+@venue_blueprint.route("/<int:venue_id>/fully-sync", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_PRO_ENTITY)
+def fully_sync_venue(venue_id: int) -> utils.BackofficeResponse:
+    venue = get_venue(venue_id)
```

Comment:
Ici tu lances donc une requête avec pas mal de jointures, alors que tu pourrais réduire puisque tu ne réaffiches pas la page. De plus, il semble manquer `VenueProvider.isActive` dans le `load_only`, ce qui déclenche a priori une requête en plus à chaque itération de la ligne en-dessous. Je propose ce code **non testé** :
```
    venue = (
        offerers_models.Venue.query.filter(offerers_models.Venue.id == venue_id)
        .options(
            .load_only(offerers_models.Venue.name, offerers_models.Venue.publicName, offerers_models.Venue.id),
            sa.orm.joinedload(offerers_models.Venue.venueProviders)
            .load_only(providers_models.VenueProvider.isActive)
        )
        .one_or_none()
    )
    
    if not venue:
        raise NotFound()
```